Android/Guava/source/Guava.FailureAccess/Guava.FailureAccess.csproj

```diff
@@ -20,38 +20,30 @@
     <Title>Google Guava FailureAccess reference library for Xamarin.Android</Title>
     <Summary>Xamarin.Android bindings for Google Guava FailureAccess</Summary>
     <Description>
-    NOTE: This package is meant to be used as a reference only for other binding projects that depend on Guava.
-    This package does not expose any useful API's in the .dll itself.
-    
-    Guava is a set of core libraries that includes new collection types (such as multimap and multiset), immutable collections, a graph library, functional types, an in-memory cache, and APIs/utilities for concurrency, I/O, hashing, primitives, reflection, string processing, and much more!
+NOTE: This package is meant to be used as a reference only for other binding projects that depend on Guava.
+This package does not expose any useful API's in the .dll itself.
+
+Guava is a set of core libraries that includes new collection types (such as multimap and multiset), immutable collections, a graph library, functional types, an in-memory cache, and APIs/utilities for concurrency, I/O, hashing, primitives, reflection, string processing, and much more!
     </Description>
     <Authors>Microsoft</Authors>
     <Owners>Microsoft</Owners>
     <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
     <PackageProjectUrl>https://go.microsoft.com/fwlink/?linkid=865028</PackageProjectUrl>
     <PackageLicenseUrl>https://go.microsoft.com/fwlink/?linkid=865030</PackageLicenseUrl>
     <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
-    <PackageVersion>1.0.1</PackageVersion>
+    <PackageVersion>1.0.1.1</PackageVersion>
   </PropertyGroup>
 
   <ItemGroup>
-    <TransformFile Include="Transforms\Metadata.xml" />
```

Comment:
We don't need any of the transforms files since we are not actually giving any .jars to the binding generator.